Traceback (most recent call last):
  File "/usr/bin/fragment", line 19, in <module>
    from python_modules.cli_base import pasta_de_seguranca, pasta_de_fragmentos, arquivo_usuarios, rclone_drive
ImportError: cannot import name 'rclone_drive'



Traceback (most recent call last):
  File "/usr/bin/sendeml", line 92, in <module>
    main()
  File "/home/bwb0de/Devel/sps_cli/cli/python_modules/cli_decorators.py", line 33, in wrapper
    return function(*args)
  File "/usr/bin/sendeml", line 58, in main
    msg['From'] = profissional['eml']
KeyError: 'eml'


Traceback (most recent call last):
  File "/usr/bin/novo", line 339, in <module>
    cmd = main()
  File "/opt/sps_cli/cli/python_modules/cli_decorators.py", line 33, in wrapper
    return function(*args)
  File "/usr/bin/novo", line 281, in main
    atd_t = novo_atendimento(identificador)
  File "/usr/bin/novo", line 164, in novo_atendimento
    nfo = render_form_get_values(formulario_atendimentos, ['identificador'])
  File "/opt/sps_cli/cli/python_modules/cli_tools.py", line 188, in render_form_get_values
    q_response = objective_question_handler(q)
  File "/opt/sps_cli/cli/python_modules/cli_tools.py", line 171, in objective_question_handler
    save_json(form, form_file)
  File "/opt/sps_cli/cli/python_modules/py_functions_json.py", line 67, in save_json
    with open(path_to_file, 'w') as f:
PermissionError: [Errno 13] Permission denied: '/opt/sps_cli/cli/formularios/form_atendimento.json'

